persistence/sql/migratest/testdata/20220902141902_testdata.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO identity_verification_tokens (id, token, used, used_at, identity_verifiable_address_id, selfservice_verification_flow_id, created_at, updated_at, expires_at, issued_at, nid)
```

Comment:
Please add another row where the column that you changed (selfservice_verification_flow_id) has a NULL value, so that we see if the migratino does what it should (i.e. not create the snapshot because the row was deleted) :) Or is that already the case?

Reply:
That fails with `Error 1048: Column 'selfservice_verification_flow_id' cannot be null
` which then fails the whole test suite.

Reply:
You mean, so that we can see, that the migration actually works on existing data? We have initial data from when this feature was initially added [here](https://github.com/ory/kratos/blob/db8c345f52670c8a2ec2181e62c638c063e73a7a/persistence/sql/migratest/testdata/20200830154602_testdata.sql). Should I add another one just in case?

Reply:
Nope, thank you for the reference - that is enough!